### ISSUE TITLE
different projections

### ISSUE THREAD

**Issue by MannusEtten** [AnalyticalGraphicsInc/cesium#3877]
i assume the cesium globe needs 4326 as projection for displaying data.

i tried one ArcGIS service which has a different projection and is not drawn on the cesium globe. I did the same exercise with a service in 4326 and perfectly drawn around the globe.

my conclusion is that the projections matters and of course it does.
does the Cesium.ArcGisMapServerImageryProvider has a property where i can set the output spatial reference? I assume the Cesium object uses exportmap-function for retrieving the picture to be drawn on the globe.

In the Esri ArcGIS Rest interface there is a property for exportmap (imageSR) where you can set the requested spatial reference. I think if i can use this property inside the Cesium-object the issue I have right now will be resolved.

**Comment by hpinkos** [comment]
Hello @MannusEtten 
I'm not exactly sure what problem you are seeing.  We recently fixed a bug for `ArcGisMapServerImageryProvider` where using a web mercator projection was resulting in a crash.  This will be fixed in the next release.
Is that the problem you are having?
Thanks!

**Comment by MannusEtten** [comment]
no that is not the problem, the problem is pretty easy:

- use a mapservice without a web mercator or wgs84 projection and there is nothing to see on the cesium-globe

**Comment by hpinkos** [comment]
@MannusEtten Cesium currently only supports web mercator and WGS84.  Do you have a way to re-project your imagery?  Projecting it to WGS84 would be ideal.

**Comment by pjcozzi** [comment]
CC #2084 and #3824

**Comment by pjcozzi** [comment]
EPSG:3395 was requested in #2084

**Comment by pjcozzi** [comment]
OSGB36 was requested in #3824

**Comment by OmarShehata** [comment]
There seems to be a lot of interest in different projections/ellipsoids lately. 

* GRS80 Ellipsoid and EPSG 2180 [on the forum](https://groups.google.com/d/msg/cesium-dev/DsIBdDJSVog/U-N-ruNQBwAJ).
* ETRS89 [on the forum](https://groups.google.com/d/msg/cesium-dev/O0SiKOJR2-8/RCxOPlt3BwAJ).
* EPSG:4490 [on the forum](https://groups.google.com/d/msg/cesium-dev/Klk4-WrtOmU/GKlRo8WzBQAJ).

**Comment by nmtoken** [comment]
> Cesium currently only supports web mercator and WGS84

Can you clarify what is meant here by **WGS84**?

WGS84 is an ellipsoid, not a projection.

Does it mean any projection that uses the WGS84 ellipsoid? or does it mean a specific projection that uses that ellipsoid, such as [epsg:4326](https://www.epsg-registry.org/export.htm?wkt=urn:ogc:def:crs:EPSG::4326)?

**Comment by OmarShehata** [comment]
@nmtoken I think that means specifically EPSG:4326 but @hpinkos can correct me if I'm wrong.

**Comment by GatorScott** [comment]
Cesium supports map image data in Web Mercator or Plate Carree/Equirectangular projections.  This issue captures community interest in having native support for other projections.

Coordinate data must be provided in x/y/z (lon/lat/height) order.

**Comment by nmtoken** [comment]
> Coordinate data must be provided in x/y/z (lon/lat/height) order.

Thanks@GatorScott so explicitly that's not EPSG:4326

**Comment by OmarShehata** [comment]
Came up again [on the forum for displaying imagery in EPSG:26917 (NAD83 / UTM zone 17N)](https://groups.google.com/forum/#!topic/cesium-dev/zmdb2V5mxoI)

**Comment by OmarShehata** [comment]
WKID 4490 requested here https://github.com/AnalyticalGraphicsInc/cesium/issues/8522.

**Comment by ggetz** [comment]
Another request for EPSG 4490: https://github.com/CesiumGS/cesium/issues/10782

**Comment by ggetz** [comment]
A request for Lambert Conformal Conic Projection in #11096.

**Comment by abigstupidbear** [comment]
seven years have not solved！

**Comment by MangMax** [comment]
I'd like to know how this issue is currently progressing and if the EPSG 4490 can be supported, which would be a great help for my current project, thanks!

**Comment by ggetz** [comment]
There hasn't been any activity on this item recently. But thank you for your input!

**Comment by ggetz** [comment]
ESPG:3857 requested in https://github.com/CesiumGS/cesium/issues/11428.

**Comment by ayacoub1** [comment]
EPSG:2932
QND95 / Qatar National Grid

**Comment by alaasquan** [comment]
I'm also having issue to render data that has EPSG 3879 in cesium environment; can someone help? My data is gLtf Tiles format .

**Comment by Zhulongtao6** [comment]
> Cesium 目前仅支持 Web Mercator 和 WGS84。你有办法重新投影你的图像吗？将其投影到 WGS84 将是理想的选择。

Hello, do you know which version of WGS 84 is used in Cesium? (WGS 84 (G730), WGS 84 (G873), WGS 84 (G1150), or WGS 84 (G1674))

**Comment by kring** [comment]
> Hello, do you know which version of WGS 84 is used in Cesium? (WGS 84 (G730), WGS 84 (G873), WGS 84 (G1150), or WGS 84 (G1674))

Those different versions of WGS84 differ only in _datum_. That is, how real-world positions are mapped to coordinates. The equations for transforming longitude/latitude/height to 3D Cartesian geocentric coordinates are unchanged, and those are the only transformations that CesiumJS does itself. So basically, the specific realization of WGS84 that CesiumJS uses is _whichever one is used by the data you're asking CesiumJS to show_. If you're using data from multiple realizations, then CesiumJS's coordinate system can be thought of as an ensemble of multiple realizations.

**Comment by Zhulongtao6** [comment]
Thank you for your prompt reply. This also means that my data source must be in one of the WGS 84 coordinate systems. If I use data from other coordinate systems, such as BJZ54, Cesium doesn't support it, right?


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "CesiumGS/cesium"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年1月14日(星期二) 晚上7:42
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [CesiumGS/cesium] different projections (#3877)





  
Hello, do you know which version of WGS 84 is used in Cesium? (WGS 84 (G730), WGS 84 (G873), WGS 84 (G1150), or WGS 84 (G1674))
  
Those different versions of WGS84 differ only in datum. That is, how real-world positions are mapped to coordinates. The equations for transforming longitude/latitude/height to 3D Cartesian geocentric coordinates are unchanged, and those are the only transformations that CesiumJS does itself. So basically, the specific realization of WGS84 that CesiumJS uses is whichever one is used by the data you're asking CesiumJS to show. If you're using data from multiple realizations, then CesiumJS's coordinate system can be thought of as an ensemble of multiple realizations.

**Comment by kring** [comment]
Right, you'll generally need to transform data in other projections / coordinate systems before bringing it into CesiumJS. The exception is certain coordinate systems that are substantially similar to WGS84 (aside from datum). An example of this is Australia's GDA2020. While it uses the GRS80 ellipsoid, that's indistinguishable from the WGS84 ellipsoid for visualization purposes. So if all your data is GDA2020, you can use it directly in CesiumJS and you effectively have a GDA2020 globe.
